tracing-subscriber/src/fmt/mod.rs

```diff
@@ -94,6 +94,29 @@
 //! // Note this will only fail if you try to set the global default
 //! // subscriber multiple times
 //! ```
+//!
+//! ### Composing Layers
+//!
+//! Composing an EnvFilter Layer and a format Layer:
```

Comment:
maybe linkify these?
```suggestion
//! Composing an [`EnvFilter`] `Layer` and a [format `Layer`](../fmt/struct.Layer.html):
```